src/dmd/backend/outbuf.d

```diff
@@ -83,38 +82,45 @@ struct Outbuffer
      */
     void reserve(size_t nbytes)
     {
+        // non-inline function for the heavy/infrequent reallocation case
+        @trusted static void enlarge(ref Outbuffer b, size_t nbytes)
+        {
+            pragma(inline, false);  // do not inline slow path
+
+            if (b.buf is null)
+            {
+                // Special-case the overwhelmingly most frequent situation
+                if (nbytes < 64) nbytes = 64;
```

Comment:
When it's confined to one line, making it an enum is actually counter-productive.

Reply:
I forgot to mention it, this is in pair with my next comment.

Reply:
The if-statement should be split up into two lines, otherwise you can't tell if the branch is covered by unittests (the line shows up green even if the branch is never taken).
Also D-style says "one statement per line" but I'm not pedantic about that.

Reply:
I'll split the line. I think we're okay without a symbolic constant.